### PR TITLE
Implemented basic structure for chrome extension

### PR DESCRIPTION
**Pull request by Faradey27**
- created basic structure
 - added ability to save/unsave article
 - added request which send data to server
 - created draft version of docs and extension build process

### CODE REVIEW

**Comment by Faradey27** [comment]
@rasendubi

**Comment by rasendubi** [comment]
Thank you!

It's better to separate your changes into multiple commits in the future, so they are more reviewable and understandable.

File: app/constants/Api.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const API_URL = 'http://localhost:5000/api/v1';
+export const ANALYTYC_USER_ID = 'UA-00000000-0';
```
Comment (rasendubi) [inline app/constants/Api.js:2]: Should be ANALYTIC

File: app/constants/EntityTypes.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export const APP = 'ARTICLE';
```
Comment (rasendubi) [inline app/constants/EntityTypes.js:1]: Just curious. Why `APP = 'ARTICLE'`?

File: app/utils/analytic.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+const defaultConfig = process.env.NODE_ENV === 'production' ? {} : {
+  options: {
+    cookieDomain: 'none',
+  },
+};
+
+const eventsHash = {
+};
+
+class Analytyc {
```
Comment (rasendubi) [inline app/utils/analytic.js:18]: Analytic

File: app/sagas/entitie.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable func-style*/
+
+import { takeEvery } from 'redux-saga';
+
+function *handleSomething() {
+  console.info('HANDLERRR');
+  yield null;
+}
+
+export function *entitie() {
```
Comment (rasendubi) [inline app/sagas/entitie.js:10]: Shouldn't this be `entity`? (as well as file name)

File: lang/messages.pot [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (rasendubi) [inline lang/messages.pot:4]: These lines are supposed to be replaced.

File: app/constants/Api.js (hunk 1 above)
Comment (Faradey27) [reply app/constants/Api.js:2]: yep

File: app/constants/EntityTypes.js (hunk 2 above)
Comment (Faradey27) [reply app/constants/EntityTypes.js:1]: mistake

File: app/sagas/entitie.js (hunk 4 above)
Comment (Faradey27) [reply app/sagas/entitie.js:10]: yep

**Pull request merged**